### PR TITLE
Build as Dynamic module support; sockets; fixes

### PR DESCRIPTION
**Pull request by cryptofuture**
What pull request includes.
1. Build as Dynamic module support for ngx_postgres (very old checks removed also)
2. Less raw nginx 1.11.6+ support. More accurate fix.
3. Backport of pull [#24](https://github.com/FRiCKLE/ngx_postgres/pull/24) request. Thats sockets support. 
I personally maintain [nginx-hda-bundle](https://github.com/cryptofuture/nginx-hda-bundle), and I want to switch it to your repository. 
Main ngx_postgres repository ignores pull request for years, and current guy who "moderate it" is very stubborn.
Basically, I also want to make your readme less scary, if its possible.

### CODE REVIEW

**Comment by Inviz** [comment]
Hello Friend. I'm sorry for a bit dated answer. I took a hiatus from this project while looking for job, but I'm thinking of getting back again soon. Thank you for understanding.

First thing first, I think dynamic loaded modules is a great idea, I never had enough time to port my modules to that stuff. So have my thanks for poking me, I'm in for it.

I appreciate you considering my fork more active-ish, however some peeps may object to features I added to ngx_postgres, but there's a decent bunch of good ones too. I tried to avoid contamination of things, so the module would be suitable for general public. So I think what you want is possible, I could clean up README by now, as I think there's nothing too untested or not ready for production. No reason that I know of to warn people about stuff.

Sockets are a great addition, thank you.

**Comment by Inviz** [comment]
I attempted to make readme less scary and mentioned your additions of socket/dynamic module loading.

If you will have any ideas about collaboration or something feel free to drop me an email at invizko@gmail.com .

Your help is very well appreciated, have a wonderful time.

**Comment by Inviz** [comment]
I also added you as a collaborator just in case.
